sky/tools/sky_utils.py

```diff
@@ -190,6 +198,16 @@ def _dsymutil_path():
   return buildroot_relative_path(dsymutil_path)
 
 
+def get_framework_name(framework_dir):
+  """Returns Foo given /path/to/Foo.framework."""
+  return os.path.splitext(os.path.basename(framework_dir))[0]
+
+
+def get_framework_dylib_path(framework_dir):
```

Comment:
Should this be `get_macos_framework_dylib_path` since `Versions/A/Foo` is macOS-specific, and sky_utils is used by both?

Reply:
Good call - yes.